config/redirects

```diff
@@ -18,4 +18,8 @@ raw: docs/atlas/cli/stablecommand/atlas-privateEndpoints-onlineArchive-aws-descr
 
 # Atlas CLI Version 1.6 and later
 
-[v1.6-*]: docs/atlas/cli/${version}/cluster-config-file -> ${base}/${version}/reference/json/cluster-config-file
\ No newline at end of file
+[v1.6-*]: docs/atlas/cli/${version}/cluster-config-file -> ${base}/${version}/reference/json/cluster-config-file
+
+# Atlas CLI Version 1.13 and earlier
+
+[*-v1.13]: /docs/atlas/cli/v1.14/atlas-cli-deploy-docker/ -> /docs/unavailable-version/
```

Comment:
```suggestion
[*-v1.13]: docs/atlas/cli/${version}/atlas-cli-deploy-docker/ -> https://www.mongodb.com/docs/unavailable-version/
```
we need a "version" in the source to use more than one version in the redirect. In the target, we need to either hard-code the URL to the unavailable version page or to add a new variable for the root docs path.